apps/api/test/setup/index.js

```diff
@@ -162,7 +162,7 @@ async function main() {
   const claimHandleProof = {
     Sr25519: u8aToHex(alice.sign(u8aWrapBytes(claimHandlePayload.toU8a()))),
   };
-  const claimHandle = api.tx.handles.claimHandle(
+  const claimHandle = api.tx.handles.changeHandle(
```

Comment:
I think this needs to be changed back for `claimHandle` for setup. 
```suggestion
  const claimHandle = api.tx.handles.claimHandle(
```